src/middlewared/middlewared/plugins/disk_/identify_freebsd.py

```diff
@@ -108,7 +108,20 @@ def identifier_to_device(self, ident, geom_scan=True, geom_xml=None):
             else:
                 xml = geom.class_by_name('DISK').xml
 
-            _ident, _lunid = value.split('_')
+            info = value.split('_')
+            info_len = len(info)
+            if info_len < 2:
+                # nothing to do return
+                return
+            elif info_len == 2:
+                _ident = info[0]
+                _lunid = info[1]
+            else:
+                # vmware nvme disks look like `VMware NVME_0000_a9d1a9a7feaf1d66000c296f092d9204`
+                # so we need to account for it
+                _lunid = info[-1]
+                _ident = info[:-info_len].rstrip('_')
```

Comment:
@yocalebo i am not sure i follow the change, it errors out ?
```
>> value = 'VMware NVME_0000_a9d1a9a7feaf1d66000c296f092d9204'
>> info = value.split('_')
>> info_len = len(info)
>> info[:-info_len].rstrip('_')
Traceback (most recent call last):
  File "<input>", line 1, in <module>
AttributeError: 'list' object has no attribute 'rstrip'
```

Reply:
A typo which I'll fix